### ISSUE TITLE
[Feature Request] Cyber Security Quiz App with Gemini LLM

### ISSUE THREAD

**Issue by J-B-Mugundh** [Vinay-Khanagavi/Cyberbear.github.io#26]
Description: 

I would like to add a quiz application and deploy the same in vercel or in github to be added as a part of this project. 

I'm a gssoc-extd and hacktoberfest contributor

**Comment by J-B-Mugundh** [comment]
@Vinay-Khanagavi Could u please assign me this issue?

**Comment by Vinay-Khanagavi** [comment]
Can you provide proper suggestion with design how that will look like... I don't want to make you work then not approve your PR

**Comment by J-B-Mugundh** [comment]
![Image](https://github.com/user-attachments/assets/69dbf0fe-0067-4a6e-9da5-d2f6f10cf77b)
![Image](https://github.com/user-attachments/assets/9fbfefe4-c434-49b5-a5f0-95aabd67f15b)

@Vinay-Khanagavi It's a basic version.. it would look like this..

**Comment by B1N4RY-P4R45173** [comment]
@J-B-Mugundh How is this different from the integrated questions that we already have? Sorry but I am not clear as of what exactly you are trying to improve. Can you join our discord server so it will be easy for all of our communications https://discord.gg/YFWmkphf

**Comment by B1N4RY-P4R45173** [comment]
Good Idea @J-B-Mugundh according to work. We will change the level if needed

**Comment by B1N4RY-P4R45173** [comment]
Hello @J-B-Mugundh just confirming if you are still working on this

**Comment by J-B-Mugundh** [comment]
> Hello @J-B-Mugundh just confirming if you are still working on this

Yeah bro! Had exams, so couldn't work much.. will raise the PR today eve

**Comment by B1N4RY-P4R45173** [comment]
No problem, take your time. Just wanted to confirm if u are working or not. So that someone else could be assigned if u are busy.
